### PR TITLE
Add information about RIDs into the controls doc

### PR DESCRIPTION
**Pull request by mmitche**
### To double check:

* [ ] The right tests are in and and the right validation has happened.  Guidance:  https://github.com/dotnet/arcade/tree/main/Documentation/Validation

### CODE REVIEW

**Comment by mmitche** [comment]
@jaredpar For docs on the TargetRid.

File: Documentation/UnifiedBuild/Unified-Build-Controls.md [hunk 1]
```diff
@@ -165,11 +165,15 @@ In addition to these default high level controls, there may be additional compon
 | BuildOS | "linux", "osx", "freebsd", "netbsd", "illumos", "solaris", "haiku", "windows", ... | Defaults to the OS of the build environment | The operating system of the machine that is built on. Lower-case string. |
 | HostOS | Same as `BuildOS` | Defaults to `BuildOS` | The operating system of the machine that will run the produced tool (i.e. compiler) to generate the binary for the target operating system. |
 | TargetOS | Same as `BuildOS` | Defaults to `BuildOS` | The operating system of the machine that will run the binary -> the end user’s machine. |
+| BuildRid | Valid RIDs | RID of the the currently executing runtime | The RID of the runtime that is running the build |
+| HostRid | Valid RIDs | Build Rid | The RID of the runtime that will run the produced tool (i.e. compiler) to generate the binary for the target operating system. |
```
Comment (ViktorHofer) [inline Documentation/UnifiedBuild/Unified-Build-Controls.md:169]: So HostArchitecture defaults to TargetArchitecture now but HostRid doesn't default to TargetRid?

Comment (mmitche) [reply Documentation/UnifiedBuild/Unified-Build-Controls.md:169]: https://github.com/dotnet/dotnet/blob/main/Directory.Build.props#L56

That said, `HostRid` is never used anywhere. I can update the default.

Comment (ViktorHofer) [reply Documentation/UnifiedBuild/Unified-Build-Controls.md:169]: Yes please update the default. Whatever direction we choose, we should be consistent between Rid and OS/Arch for the defaults.

**Review by ViktorHofer — APPROVED** [review]

**Pull request merged**